lib/tasks/application_forms.rake

```diff
@@ -61,4 +61,12 @@ namespace :application_forms do
       puts "#{application_form.reference}: #{application_form.status}"
     end
   end
+
+  desc "Update status of application forms in waiting on to overdue"
+  task :update_waiting_on_applications, :environment do
+    user = "Status updater"
+    ApplicationForm.waiting_on.each do |application_form|
```

Comment:
I think we could also filter on `ApplicationForm`s which are `waiting_on` _and_ have an expired requestable, reducing the number of application forms we need to update the status of.

Reply:
Updated =)